### PR TITLE
[docs] Update ctc_loss function parameters in documentation

### PR DESCRIPTION
**Pull request by aztice**
### 描述

将 `norm_by_times` 以及 `zero_infinity` 同步于 `ctc_loss` 函数
> 详见 https://github.com/PaddlePaddle/Paddle/pull/75742 and PaddlePaddle/Paddle#76156

### CODE REVIEW

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst:1]: `paddle.nn.CTCLoss` 不用改么？

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst:1]: 完成

**Review by mattheliu — APPROVED** [review]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md
Comment (SigureMo) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: 还是「torch 参数更多」吗？

Comment (aztice) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: okk抱歉喵没看到。

Comment (SigureMo) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: `CTCLoss` 呢？不用删吗？

Comment (aztice) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: image
？还有什么要删除的吗

Comment (SigureMo) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: 为什么不改 docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.functional.ctc_loss.md ？

Comment (aztice) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CTCLoss.md:1]: okk，我这边全局搜索漏了ctc_loss抱歉呐

**Comment by luotao1** [comment]
@SigureMo 请再看下

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ ctc_loss
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
     - **norm_by_times** (bool，可选) - 是否根据序列长度对梯度进行正则化。数据类型支持 bool。默认值为 False。
+    - **zero_infinity** (bool，可选） - 防止梯度出现无限，将其重置为 0，以免训练时发生爆炸。
```
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst]:
```suggestion
    - **zero_infinity** (bool，可选） - 防止梯度出现无限，将其重置为 0。
```

英文没有这句

File: docs/api/paddle/nn/CTCLoss_cn.rst [hunk 2]
```diff
@@ -12,6 +12,7 @@ CTCLoss
 :::::::::
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
+    - **zero_infinity** (bool，可选） - 防止梯度出现无限，将其重置为 0，以免训练时发生爆炸。
```
Comment (SigureMo) [inline docs/api/paddle/nn/CTCLoss_cn.rst]: 同上

Comment (SigureMo) [reply docs/api/paddle/nn/CTCLoss_cn.rst]: 英文文档是不是没加这个参数？

Comment (aztice) [reply docs/api/paddle/nn/CTCLoss_cn.rst]: 英文文档在哪里emm我找不到（

Comment (SigureMo) [reply docs/api/paddle/nn/CTCLoss_cn.rst]: Paddle 那边的 docstring

Comment (aztice) [reply docs/api/paddle/nn/CTCLoss_cn.rst]: https://github.com/PaddlePaddle/Paddle/pull/76188

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst [hunk 3]
```diff
@@ -17,6 +17,7 @@ ctc_loss
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
     - **norm_by_times** (bool，可选) - 是否根据序列长度对梯度进行正则化。数据类型支持 bool。默认值为 False。
+    - **zero_infinity** (bool，可选） - 防止梯度出现无限，将其重置为 0。
```
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 说明其行为，而不是其目标，和英文文档对齐

另外右括号也应该为英文括号而非中文括号

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst]: OK

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst [hunk 4]
```diff
@@ -17,6 +17,7 @@ ctc_loss
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
     - **norm_by_times** (bool，可选) - 是否根据序列长度对梯度进行正则化。数据类型支持 bool。默认值为 False。
+    - **zero_infinity** (bool，可选) - 是否启用零冗余优化器。
```
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 翻译这句话 `If True, set infinite loss to zero. Default: False.`

「零冗余优化器」是什么？

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 明白。

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst [hunk 5]
```diff
@@ -17,6 +17,7 @@ ctc_loss
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
     - **norm_by_times** (bool，可选) - 是否根据序列长度对梯度进行正则化。数据类型支持 bool。默认值为 False。
+    - **zero_infinity** (bool，可选) - 如果 ``true``，将 可能 输出无限的 ``loss`` 设置为 ``0``。默认值: False。
```
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 啊……

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 啊？

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 我感觉我平时写md或者这类文档的时候习惯比较awful（）

Comment (aztice) [reply docs/api/paddle/nn/functional/ctc_loss_cn.rst]: 会打空格进去特别标注（

File: docs/api/paddle/nn/functional/ctc_loss_cn.rst [hunk 6]
```diff
@@ -17,6 +17,7 @@ ctc_loss
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
     - **norm_by_times** (bool，可选) - 是否根据序列长度对梯度进行正则化。数据类型支持 bool。默认值为 False。
+    - **zero_infinity** (bool，可选) - 如果 ``true``，将可能输出无限的 ``loss`` 设置为 ``0``。默认值: False。
```
Comment (SigureMo) [inline docs/api/paddle/nn/functional/ctc_loss_cn.rst]:
```suggestion
    - **zero_infinity** (bool，可选) - 如果为 ``true``，则将 ``loss`` 中无限的值设置为 ``0``。默认值为 False。
```

File: docs/api/paddle/nn/CTCLoss_cn.rst [hunk 7]
```diff
@@ -12,6 +12,7 @@ CTCLoss
 :::::::::
     - **blank** (int，可选) - 空格标记的 ID 值，其取值范围为 [0, num_classes + 1)。数据类型支持 int32。默认值为 0。
     - **reduction** (str，可选) - 指定应用于输出结果的计算方式，可选值有：``'none'``, ``'mean'``, ``'sum'``。设置为 ``'mean'`` 时，对 loss 值除以 label_lengths，并返回所得商的均值；设置为 ``'sum'`` 时，返回 loss 值的总和；设置为 ``'none'`` 时，则直接返回输出的 loss 值。默认值为 ``'mean'``。
+    - **zero_infinity** (bool，可选) - 如果 ``true``，将可能输出无限的 ``loss`` 设置为 ``0``。默认值: False。
```
Comment (SigureMo) [inline docs/api/paddle/nn/CTCLoss_cn.rst]:
```suggestion
    - **zero_infinity** (bool，可选) - 如果为 ``true``，则将 ``loss`` 中无限的值设置为 ``0``。默认值为 False。
```

**Review by SigureMo — APPROVED** [review]
